src/screens/DiscoveryScreenV2/index.js

```diff
@@ -382,7 +382,7 @@ export default DiscoveryScreenV2;
 
 const DiscoveryContainer = ({children}) => {
   return (
-    <SafeAreaView style={{flex: 1, backgroundColor: 'white'}}>
+    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the `backgroundColor` of the `SafeAreaView` from 'white' to `COLORS.almostBlack` will significantly alter the appearance of your app. Ensure that all child components and text within this view are still clearly visible with this darker background color.

```diff
- <SafeAreaView style={{flex: 1, backgroundColor: 'white'}}>
+ <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->